site.yml

```diff
@@ -45,6 +45,14 @@
       tags: ['infra', 'rabbitmq']
   environment: "{{ env_vars|default({}) }}"
 
+- name: setup serverspec
+  hosts: all
+  roles:
+    - role: serverspec
+      tags: ['serverspec']
+      when: serverspec.enabled|bool
+  environment: "{{ env_vars|default({}) }}"
```

Comment:
We have it as its own play in other ansible repos ...   would be good to keep it like that for consistency....  although TBH if you want you could skip the whole serverspec play alltogether and just bolster the stuff in sensu-client to do the whole lot.  would allow you to skip the ruby install as well,  I just like having `cd /etc/serverspec && rake` available easily.